qgis/QGIS_PluginProperWay.md

```diff
@@ -0,0 +1,230 @@
+### QGIS Tutorials
+
+This tutorial is part of our QGIS tutorial series:
+
+- [QGIS 3 Plugins - Plugin 101](https://gis-ops.com/qgis-3-plugin-tutorial-plugin-development-reference-guide/)
+- [QGIS 3 Plugins - Qt Designer Explained](https://gis-ops.com/qgis-3-plugin-tutorial-qt-designer-explained/)
+- [QGIS 3 Plugins - Signals and Slots in PyQt](https://gis-ops.com/qgis-3-plugin-tutorial-pyqt-signal-slot-explained/)
+- [QGIS 3 Plugins - Plugin Development Part 1](https://gis-ops.com/qgis-3-plugin-tutorial-plugin-development-explained-part-1/)
+- [QGIS 3 Plugins - Plugin Development Part 2](https://gis-ops.com/qgis-3-plugin-tutorial-plugin-development-explained-part-2/)
+- [QGIS 3 Plugins - Plugin Development Part 3](https://gis-ops.com/qgis-3-plugin-tutorial-plugin-development-explained-part-3/)
+- [QGIS 3 Plugins - Set up Plugin Repository](https://gis-ops.com/qgis-3-plugin-tutorial-set-up-a-plugin-repository-explained/)
+
+
+---
+
+# QGIS 3 Plugins - Apply Best Practices & Useful Tips
```

Comment:
Generally we could use some titles that are a little more catchy... Geocoding with nominatim / opencage etc...? What do you think?

Reply:
Yep, sounds good. Then `QGIS 3 Plugins - Geocoding with Nominatim Part 1` - 3.